stdlib/2and3/genericpath.pyi

```diff
@@ -7,16 +7,21 @@ else:
     def commonprefix(m: Sequence[AnyStr]) -> AnyStr: ...
 if sys.version_info >= (3, 6):
     from builtins import _PathLike
-    def exists(path: Union[AnyStr, _PathLike[AnyStr]]) -> bool: ...
+    def exists(path: Union[AnyStr, _PathLike[AnyStr], int]) -> bool: ...
+    def isfile(path: Union[AnyStr, _PathLike[AnyStr], int]) -> bool: ...
+    def isdir(s: Union[AnyStr, _PathLike[AnyStr], int]) -> bool: ...
+    def getsize(filename: Union[AnyStr, _PathLike[AnyStr], int]) -> int: ...
+    def getmtime(filename: Union[AnyStr, _PathLike[AnyStr], int]) -> float: ...
+    def getatime(filename: Union[AnyStr, _PathLike[AnyStr], int]) -> float: ...
+    def getctime(filename: Union[AnyStr, _PathLike[AnyStr], int]) -> float: ...
 else:
     def exists(path: Text) -> bool: ...
-def isfile(path: Text) -> bool: ...
-def isdir(s: Text) -> bool: ...
-def getsize(filename: Text) -> int: ...
-def getmtime(filename: Text) -> float: ...
-def getatime(filename: Text) -> float: ...
-def getctime(filename: Text) -> float: ...
-
+    def isfile(path: Text) -> bool: ...
+    def isdir(s: Text) -> bool: ...
+    def getsize(filename: Text) -> int: ...
+    def getmtime(filename: Text) -> float: ...
+    def getatime(filename: Text) -> float: ...
+    def getctime(filename: Text) -> float: ...
```

Comment:
I would probably define an alias _Path, depending on the Python version, and then avoid duplication by just defining each function separately like this:

```python
def exist(path: _Path) -> bool: ...
# etc.
```